.github/workflows/build.yml

```diff
@@ -46,12 +46,16 @@ jobs:
 
       - name: Build sample app
         working-directory: /tmp/testapp
-        run: ./gradlew assemble -PclientSpecific=true
+        run: ./gradlew build -PclientSpecific=true
 
       - name: Skip binding to privileged port on CI
         working-directory: /tmp/testapp
         run: sed -i "s/- '443:443'/# - '443:443'/g" docker-compose.yml
 
+      - name: Create Server Dockerfile
+        working-directory: /tmp/testapp
+        run: ./gradlew :genesisproduct-testapp:testapp-deploy:createDockerfile
```

Comment:
does "testapp" need to be templated?

Reply:
No, this is just for the github build. Creates a new app, hardcoded to testapp, and builds it

Reply:
Looks like the .github folder is excluded from generated projects so is hardcoded here for testing